<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="output_canvas1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>75</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pitch_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>680</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Pitch(x)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="yaw_button">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>680</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Yaw(y)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="roll_button">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>680</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Roll(z)</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="rotateStepDoubleSpinBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>710</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="keyboardTracking">
    <bool>true</bool>
   </property>
   <property name="minimum">
    <double>-99.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.200000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="leftMoveButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>680</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>left</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rightMoveButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>710</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>right</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upMoveButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>680</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="downMoveButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>710</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>down</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="moveStepDoubleSpinBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>680</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="keyboardTracking">
    <bool>true</bool>
   </property>
   <property name="minimum">
    <double>-99.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="aheadMoveButton">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>680</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>ahead</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backMoveButto">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>710</y>
     <width>51</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
  <widget class="QLabel" name="output_canvas2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>150</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel2</string>
   </property>
  </widget>
  <widget class="QLabel" name="output_canvas3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>225</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel3</string>
   </property>
  </widget>
  <widget class="QLabel" name="output_canvas4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel4</string>
   </property>
  </widget>
  <widget class="QLabel" name="output_canvas5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>375</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel5</string>
   </property>
  </widget>
  <widget class="QLabel" name="output_canvas6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel6</string>
   </property>
  </widget>
  <widget class="QLabel" name="output_canvas7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>525</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel7</string>
   </property>
  </widget>
  <widget class="QLabel" name="output_canvas8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel8</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="lightingCheckBox">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>670</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>lighting</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="diffuseCheckBox">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>670</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>diffuse</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="mirrorCheckBox">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>690</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>mirror</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="shadowCheckBox">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>690</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>shadow</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="multithreadsCheckBox">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>710</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>multithreads</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="renderPushButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>730</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>render</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>490</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="timeTestPushButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>520</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>time test</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>490</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>мс</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>400</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Измеряется время&lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;для четырех изменений &lt;/p&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;положения камеры&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="reflectionCheckBox">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>710</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>reflection</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="cameraRadioButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>610</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Camera</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="pointLightRadioButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>630</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Point light</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="directionLightRadioButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>650</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Direction light</string>
   </property>
  </widget>
  <zorder>pitch_button</zorder>
  <zorder>yaw_button</zorder>
  <zorder>roll_button</zorder>
  <zorder>rotateStepDoubleSpinBox</zorder>
  <zorder>leftMoveButton</zorder>
  <zorder>rightMoveButton</zorder>
  <zorder>upMoveButton</zorder>
  <zorder>downMoveButton</zorder>
  <zorder>moveStepDoubleSpinBox</zorder>
  <zorder>aheadMoveButton</zorder>
  <zorder>backMoveButto</zorder>
  <zorder>output_canvas8</zorder>
  <zorder>output_canvas7</zorder>
  <zorder>output_canvas6</zorder>
  <zorder>output_canvas5</zorder>
  <zorder>output_canvas4</zorder>
  <zorder>output_canvas3</zorder>
  <zorder>output_canvas2</zorder>
  <zorder>output_canvas1</zorder>
  <zorder>lightingCheckBox</zorder>
  <zorder>diffuseCheckBox</zorder>
  <zorder>mirrorCheckBox</zorder>
  <zorder>shadowCheckBox</zorder>
  <zorder>multithreadsCheckBox</zorder>
  <zorder>renderPushButton</zorder>
  <zorder>label</zorder>
  <zorder>timeTestPushButton</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>reflectionCheckBox</zorder>
  <zorder>cameraRadioButton</zorder>
  <zorder>pointLightRadioButton</zorder>
  <zorder>directionLightRadioButton</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
